README.md

```diff
@@ -23,19 +35,64 @@ The #1 best way to understand and address the weaknesses in an API's design and
+These standards assuming the APIs will be generally "Restful". However, recommendations are also provided for SOAP web services.
+) to understand the developer experience in using your API.
+
+### Add Your API To The GSA API Directory
+A directory of GSA public APIs is available at [open.gsa.gov/developer](open.gsa.gov/developer/). You can add your API to this directory by posting an issue or pull request in the [GitHub repository](https://github.com/GSA/open.gsa.gov).
+
+### Provide Documentation
+The developer's entry point to your API will likely be the documentation that you provide. GSA has developed an [API Documentation Template](https://github.com/GSA/api-documentation-template) which can easily be re-used for your API.
+
+At a minimum, API documentation should provide:
+* An overview of the contents of the API and the data sources.
+* Production URLs for accessing the API.
+* Required parameters and defaults.
+* A description of the data that is returned.
+* A description of the error codes that are returned, and their meaning.
+
+Additional nice-to-haves include:
+* Explanation of key management and a sample key.
+* Description of update frequency.
+* Interactive documentation to demonstrate sample calls.
+* Sample client code for consuming the API in common languages.
+
 ### Point of contact
 
 Have an obvious mechanism for clients to report issues and ask questions about the API.
 
 When using GitHub for an API's code, use the associated issue tracker. In addition, publish an email address for direct, non-public inquiries.
 
+### Avoid an API "Ghost Town" by responding to issues and questions
+Developers can immediately sniff out a stale and unsupported API by old issues never answered and contact emails that no longer work.
+
+It is critical to respond to issues posted or queries submitted by developers. This demonstrates that the API can be counted on for production usage. If an immediate fix (or even a developer to investigate) is not readily available, respond anyway. Developers will be glad to know when you'll be able to take a look.
+
 ### Notifications of updates
 
 Have a simple mechanism for clients to follow changes to the API.
 
 Common ways to do this include a mailing list, or a [dedicated developer blog](https://developer.github.com/changes/) with an RSS feed.
 
+### Decommission Unsupported APIs
+
+If an API can no longer be supported, consider decommissioning the API and removing the documentation. If the API will remain available for historical purposes without support, update the documentation to reflect this.
+
+### Avoid Breaking Changes
+
+*TodDo: flesh this out*
+
+## Design considerations
+
 ### API Endpoints
+*Todo: give specific recommendations on URL*
```

Comment:
Yes I posted issue #8 about this, because I have a pretty good idea. I am waiting until I get a little more feedback regarding the GSA taxonomy before I add it.